slides/testing/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Continuing from the `npm test` command comment. Can we not just use a tsconfig file?
Right now it doesn't seem like this is being used